docs/customization_guide/inference_protocols.md

```diff
@@ -114,6 +114,55 @@ These options can be used to configure the KeepAlive settings:
 
 For client-side documentation, see [Client-Side GRPC KeepAlive](https://github.com/triton-inference-server/client/blob/main/README.md#grpc-keepalive).
 
+### Limit Endpoint Access
```

Comment:
Can we provide these settings in a json config file instead of CLI options? The optiion might look difficult to comprehend with protocol type, protocols(which are already quite a few and might grow in the future), port (and may be address if the user wants different address for this service).

Reply:
I am inclined to have separate config files but I don't think that is in favorite in a containerized setting, i.e. configure and start Triton in single command line without need to maintain / mount additional files. CC @Tabrizian @rmccorm4 as it is what I get out from cache dicussion.

Reply:
Maybe we could have a global configuration file that let's you express all the CLI flags (similar to MA) so that you can more easily describe nested settings? The CLI flags will be given more priority if it is described in both CLI and JSON config file.

Reply:
But we do want to keep the ability to specify via CLI alone, right? config file will be an addition instead of replacement?

Reply:
Yes, it will be an additional point of configuration.

Reply:
configuration files are fine in containers as well. we use them heavily in TMS